Repositories/ProjectRepository.cs

```diff
@@ -287,44 +228,153 @@ public override void Update(Project entity)
         }
 
         /// <summary>
-        /// Checks if any of the searchable fields of the project passed contains the provided query.
+        ///     Redact user email from the Project if isPublic setting is set to false
+        /// </summary>
+        /// <param name="project">The project.</param>
+        /// <returns>
+        ///     Project with possibly redacted email depending on setting
+        /// </returns>
+        private Project RedactUser(Project project)
+        {
+            if(project == null) return null;
+
+            if(project?.User?.IsPublic == false)
+            {
+                project.User.Email = Defaults.Privacy.RedactedEmail;
+            }
+            return project;
+        }
+
+        /// <summary>
+        ///     Redact user email from the User if isPublic setting is set to false
```

Comment:
This is an example you didn't add a dot at the end of the sentence

Reply:
Thanks, you're right. I kinda forget sometimes, which is why we have these pull requests ^^